kubeface/broadcast.py

```diff
@@ -0,0 +1,21 @@
+import hashlib
+import logging
+
+class Broadcast(object):
+    def __init__(self, storage_prefix, data):
+        hash_path = '%s/%s' % (storage_prefix, self.compute_filename(data))
+        logging.info('Writing broadcast data to %s', hash_path)
+        with open(hash_path, 'w') as f:
+            f.write(data)
+        self.path = hash_path
+        self.data = None
+
+    @staticmethod
+    def compute_filename(data):
+        return hashlib.md5(data.encode('utf-8')).hexdigest()
+
+    def value(self):
+        if self.data == None:
```

Comment:
tiny style nit: always want to use `self.data is None` instead of `self.data == None`. See second bullet point of https://www.python.org/dev/peps/pep-0008/#programming-recommendations